src/views/canvas/CanvasSeqBlock.js

```diff
@@ -131,9 +131,7 @@ const View = boneView.extend({
 
   draw: function() {
     if (!(this.g.config.get("shouldRenderSeqBlockAsSvg") === true)) {
-      // fastest way to clear the canvas
-      // http://jsperf.com/canvas-clear-speed/25
-      this.el.width = this.el.width;
```

Comment:
The original performance report doesn't exist anymore, and from quickly skimming online `clearReact` seems like the fastest and less buggy option.